src/main/java/org/mule/extension/db/internal/result/resultset/ListResultSetHandler.java

```diff
@@ -24,9 +25,16 @@
 public class ListResultSetHandler implements ResultSetHandler {
 
   private final RowHandler rowHandler;
+  private final Charset charset;
 
   public ListResultSetHandler(RowHandler rowHandler) {
```

Comment:
Same as above, are we still using this constructor somewhere?

Reply:
Yes, we are. Not all operation require the charset